modules/gdscript/gdscript.cpp

```diff
@@ -892,14 +916,15 @@ bool GDScript::_get(const StringName &p_name, Variant &r_ret) const {
 				return true;
 			}
 		}
-
 		{
 			HashMap<StringName, MemberInfo>::ConstIterator E = top->static_variables_indices.find(p_name);
 			if (E) {
 				if (E->value.getter) {
-					Callable::CallError ce;
-					r_ret = const_cast<GDScript *>(this)->callp(E->value.getter, nullptr, 0, ce);
-					return true;
+					const GDScript::MemberInfo *member = &E->value;
+					Callable::CallError ce = call_getter(*member, p_name, r_ret);
+					if (ce.error == Callable::CallError::CALL_OK) {
+						return true; //TODO: should this just be return ce.error == Callable::CallError::CALL_OK?
+					}
```

Comment:
Note that this _already_ makes a change to the logic. Also, the comment is missing a space before "TODO". The same code is below in 5 more places.

Reply:
This particular line was kind of odd. It was the only getter/setter call that didn't use the call error and just returned true no matter what. At first, I did the same thing, but the CI complained that I didn't use the returned call error. I would love if someone could explain the logic around this particular get call